helpers/contracts-deployments.ts

```diff
@@ -3276,6 +3278,37 @@ export const deployDelegationAwarePToken = async (
   return instance;
 };
 
+export const deployPapaX = async (verify?: boolean) => {
+  const implementation = await deployParaXImpl(verify);
+
+  const initData = implementation.interface.encodeFunctionData("initialize");
+
+  const proxyInstance = await withSaveAndVerify(
+    await getContractFactory("InitializableAdminUpgradeabilityProxy"),
+    eContractid.ParaX,
+    [],
+    verify
+  );
+
+  const proxyAdmin = "0x17816E9A858b161c3E37016D139cf618056CaCD4"; //yubo.eth
```

Comment:
seems hard coded

Reply:
might better be our safe address? 0x19293FBec52F94165f903708a74513Dd6dFedd0a

Reply:
It's a one-time deploy contract and yubo ask use his addrss as proxy admin